packages/iris-grid/src/IrisGridUtils.ts

```diff
@@ -1495,6 +1495,69 @@ class IrisGridUtils {
       })
       .filter(filterConfig => filterConfig != null) as [number, T][];
   }
+
+  /**
+   * @param columnType The column type that the filters will be applied to.
+   * @param filterList The list of filters to be combined.
+   * @returns The combination of the filters in filterList as text.
+   */
+  static combineFiltersFromList(
+    columnType: string,
+    filterList: FilterData[]
+  ): string {
+    filterList.sort((a, b) => {
+      // move all 'equals' comparisons to end of list
+      const aOperator =
+        a.value == null && a.operator !== 'notEq' ? 'eq' : a.operator;
+      const bOperator =
+        b.value == null && b.operator !== 'notEq' ? 'eq' : b.operator;
+      if (aOperator === 'eq' && bOperator !== 'eq') {
+        return 1;
+      }
+      if (aOperator !== 'eq' && bOperator === 'eq') {
+        return -1;
+      }
+      return a.startColumnIndex - b.startColumnIndex;
+    });
+
+    let combinedText = '';
+    for (let i = 0; i < filterList.length; i += 1) {
+      let { operator } = filterList[i];
+      const { text, value } = filterList[i];
+      if (value !== undefined) {
+        let symbol = '';
+        if (operator !== undefined) {
+          if (value == null && operator !== 'notEq') {
+            symbol = '=';
```

Comment:
I don't think this is right. If I have the `>` or `<` operator selected, and then double click a null value, I'm getting a filter `=null` which doesn't seem right? Or I guess that's intentional and what @dsmmcken asked for?
If we're doing that for a reason, should put a comment here. And with the test case that tests that behaviour as well, it doesn't seem intuitive to me (I would've expected `>null` which would be invalid of course, so then just dropping that part of the filter...)

Reply:
Confirming behaviour should be as bender said, put the invalid link through so it visually fails to the user.